### ISSUE TITLE
Add Meson Build System Support to ddX

### ISSUE THREAD

**Issue by lukaswittmann** [ddsolvation/ddX#155]
We are currently working on the implementation of ddX into our [tblite](https://github.com/tblite/tblite) and [xTB](https://github.com/grimme-lab/xtb) program packages.
As the [meson](https://github.com/grimme-lab/xtb/blob/main/meson/README.adoc) build system is our main workhorse for compiling, we would need this to be available for ddX.

Is it planned in the near future to incorporate meson? If not, we will look into implementing it ourselves via a PR.

CC: @robin-dahl

**Comment by lukaswittmann** [comment]
We are currently already implementing it, it can be found in [my fork](https://github.com/lukaswittmann/ddX/tree/meson).

**Comment by mnottoli** [comment]
No, it was not a plan for the immediate future. So thanks for working on it. Please let me know if you need any assistance.

**Comment by lukaswittmann** [comment]
A pull request is now open (#157) that implements the Meson build system for ddX.
